README.md

```diff
@@ -37,6 +37,7 @@ Shortenator.configure do |config|
   config.bitly_token = ENV['BITLY_TOKEN']
   config.remove_protocol = true # OPTIONAL false by default
   config.tags = ['repo_name'] # OPTIONAL empty by default, let you auto tag all bit.ly links for organization
+  config.group_guid = ENV['DEFAULT_BITLY_GROUP_GUID'] #OPTIONAL the bitly docs recommend to set this in the event you accidently switch your token's group and hit a smaller limit than you intended. source: https://dev.bitly.com/v4/#operation/createFullBitlink
```

Comment:
This seems bitly-specific, so would calling this `bitly_group_guid` be more precise? The `tags` seems like something that could reasonable be generic between backends. The other idea would be to name this something _more_ generic like `group_id`. Then again, this probably doesn't matter.